README.md

```diff
@@ -43,6 +43,8 @@ equivalent to running `./configure` with autotools.
     $ cmake ../
 ```
 
+Note : The test build are by default `ON`, you can avoid building test by providing args to CMake as `cmake -DBUILD_TESTS=OFF ../`
```

Comment:
I think we should also add some doc mentioning about how to run tests using `./mlboard_tests`. in build directory.